CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Release Notes
 
+## Version 1.10.0-beta.0 - 2020-04-09
+Changes:
+* fallback to alternate drivers on failure [#7389](https://github.com/kubernetes/minikube/pull/7389)
+* start: Parallelize mount, cache push, and addon enablement [#7525](https://github.com/kubernetes/minikube/pull/7525)
+* Extract preloaded tarball as soon as minikube volume is created (2.5s speedup) [#7490](https://github.com/kubernetes/minikube/pull/7490)
+* Update error message for the user UI in start.go [#7500](https://github.com/kubernetes/minikube/pull/7500)
```

Comment:
Remove changes which users don't need to take heed of, like this one -- and the performance improvement ones.